Project2015To2017/Reading/ConditionEvaluator.cs

```diff
@@ -0,0 +1,172 @@
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Project2015To2017.Reading.Conditionals;
+
+namespace Project2015To2017.Reading
+{
+	public static class ConditionEvaluator
+	{
+		#region MSBuild Conditional routine
+
+		private static readonly Lazy<Regex> SinglePropertyRegex = new Lazy<Regex>(
```

Comment:
The cost of initializing this is not worth the Lazy<T> construct?

Reply:
MSBuild code again (probably to make no-op actions faster). But this is indeed sth I could change. Will do.